src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,67 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+ellipse:hover,rect:hover,path:hover,text:hover,linearGradient:hover,polygon:hover,circle:hover {
```

Comment:
this might also work, just try it out:
`ellipse,rect,path,text,linearGradient,polygon,circle{
&:hover{
}
}
`

Reply:
is this an ES6 its looks fancy lol

Reply:
I changed it its works also.

Reply:
awesome, this is syntax specific to scss maybe you can use it in something else you write in the future